packages/api/src/lib/granule-delete.ts

```diff
@@ -97,7 +97,12 @@ const deleteGranuleAndFiles = async (params: {
     logger.debug(`Successfully deleted granule ${apiGranule.granuleId} from ES datastore`);
     await deleteS3Files(apiGranule.files);
     logger.debug(`Successfully removed S3 files ${JSON.stringify(apiGranule.files)}`);
-    return;
+    return {
+      collection: apiGranule.collectionId,
+      deletedGranuleId: apiGranule.granuleId,
+      deletionTime: Date.now(),
+      deletedFiles: apiGranule.files,
```

Comment:
Ah I didn't realize this wasn't in feature at all. That _is_ curious why the test asserting `deletedFiles.length` passed. It should have been undefined.

Note to look into this further.